altair/vegalite/v5/schema/core.py

```diff
@@ -487,6 +487,7 @@ def load_schema() -> dict:
 
 
 class VegaLiteSchema(SchemaBase):
+    _schema = load_schema()
     _rootschema = load_schema()
```

Comment:
Linking this back to:
- [refactor: Ensure every `VegaLiteSchema` has a `._schema`](https://github.com/vega/altair/pull/3547/commits/bd31d7c574820ec3ee3bf72ec874c96545f415ba)

Has the same result for `Root`, since it inherits `_schema`.

The main benefit is a stronger guarantee that every derived class is non-None for both attributes